daal4py/sklearn/linear_model/_ridge.py

```diff
@@ -101,13 +101,11 @@ def _daal4py_predict(self, X):
     return res
 
 
-def _fit_ridge(self, X, y, sample_weight=None):
+def _fit_ridge(self, _X, _y, sample_weight=None):
     if sklearn_check_version("1.0") and not sklearn_check_version("1.2"):
         self._normalize = _deprecate_normalize(
             self.normalize, default=False, estimator_name=self.__class__.__name__
         )
-    if sklearn_check_version("1.0"):
```

Comment:
why remove?

Reply:
check_features_names is an aspect which is called within sklearn's validate_data (not sure why they did it this way in the first place) https://github.com/scikit-learn/scikit-learn/blob/1.0.X/sklearn/base.py#L552. The inclusion of validate_data later on makes the whole if statement useless (and is in a sklearn_version_check("1.0"))